integration-cli/docker_cli_daemon_test.go

```diff
@@ -481,6 +481,31 @@ func TestDaemonUpgradeWithVolumes(t *testing.T) {
 	logDone("daemon - volumes from old(pre 1.3) daemon work")
 }
 
+// GH#11320 - verify that the daemon exits on failure properly
+func TestDaemonExitOnFailure(t *testing.T) {
```

Comment:
@estesp can we add a comment saying that this will also test conflict between `-b` and `--bip` ? Also, i'd rather we use the full flag it's more descriptive. So `--bridge` instead of `-b`.